pipdeptree.py

```diff
@@ -266,6 +268,33 @@ def jsonify_tree(tree, indent):
                       indent=indent)
 
 
+def dumps_graphviz_dot(tree):
+    """Convert dependency graph to dot code for layout with graphviz.
+
+    :param dict tree: dependency graph
+    :returns: dot representation of tree
+    :rtype: str
+
+    """
+    import networkx as nx
```

Comment:
I think you should handle `ImportError` here and fail with an error message mentioning the additional dependencies for the `--dot` flag